content/docs/1.6.0/deploy/install/install-with-fleet.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Install with Fleet
+weight: 10
+---
+
+## Prerequisites
+- Your workstation: Install [Helm](https://helm.sh/docs/) v2.0 or later.
```

Comment:
Suggest the currently supported version v3 instead, because v2 was EOL.

Reply:
Refined.